examples/native/.storybook/main.js

```diff
@@ -3,5 +3,9 @@ module.exports = {
     './components/**/*.stories.?(ts|tsx|js|jsx)',
     './other_components/AnotherButton/AnotherButton.stories.tsx',
   ],
-  addons: ['@storybook/addon-ondevice-notes', '@storybook/addon-ondevice-controls'],
+  addons: [
+    '@storybook/addon-ondevice-notes',
+    '@storybook/addon-ondevice-controls',
+    '@storybook/addon-ondevice-backgrounds',
```

Comment:
It's interesting that this seems to work even though there's no explicit dependency at https://github.com/lauriharpf/react-native/blob/feat/reintroduce-backgrounds-addon/examples/native/package.json

`yarn why` says this:

```
lauriharpf@Lauris-MacBook-Pro native % yarn why @storybook/addon-ondevice-backgrounds
yarn why v1.22.10
[1/4] 🤔  Why do we have the module "@storybook/addon-ondevice-backgrounds"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "@storybook/addon-ondevice-backgrounds@6.0.0-alpha.0"
info Has been hoisted to "@storybook/addon-ondevice-backgrounds"
info Reasons this module exists
   - "workspace-aggregator-308ac265-8f7c-4c55-8d6d-d9a9373c67cf" depends on it
   - Specified in "dependencies"
   - Hoisted from "_project_#@storybook#addon-ondevice-backgrounds"
info Disk size without dependencies: "1.66MB"
info Disk size with unique dependencies: "10.8MB"
info Disk size with transitive dependencies: "34.95MB"
info Number of shared dependencies: 81
```

Should we still explicitly add the dependency to the `examples/native` project (`yarn add -D @storybook/addon-ondevice-backgrounds`) so that people don't get confused?

Reply:
Yeah it should be in the package.json in examples/native. Probably its only working because of the metro config that adds the packages to the watcher.

Reply:
Added the dependency to the examples/native package.json :+1: